src/kpro_req_lib.erl

```diff
@@ -89,6 +89,42 @@
                        , session_epoch => kpro:int32()
                        , rack_id => iodata()
                        }.
+%% Options for a `fetch' request.
```

Comment:
Put this above the type spec?

Reply:
It is a bit weird (and not consistent with how functions are documented) but in EDoc types can be documented either at the end of the same line as the definition is or, in case of a mutliline documentation, below the type definition. See the [EDdoc docs](https://www.erlang.org/doc/apps/edoc/chapter.html#type-definitions).